Source/Graph/Graph.cs

```diff
@@ -1106,6 +1106,36 @@ public ICollection<Node> Reachable()
 
   public static class GraphAlgorithms
   {
+
+    /**
+     * A merge node is a node that has multiple incoming edges, and which cannot be traversed unless all incoming edges have been traversed.
+     * A merge node is represented by an object of type IReadOnlySet{object}
+     */
+    public static IEnumerable<object> FindReachableNodesInGraphWithMergeNodes(Dictionary<object, List<object>> edges, IEnumerable<object> roots)
+    {
+      var todo = new Stack<object>(roots);
+      var visitedEdges = new HashSet<object>();
+      while(todo.Any())
+      {
+        var node = todo.Pop();
+        if (visitedEdges.Contains(node)) continue;
```

Comment:
Put the "then" branch (that is, the `continue;` statement) into curly braces.

Reply:
In the file `Source/StyleCop.ruleset` there's a line `<Rule Id="SA1503" Action="None" /> <!-- Braces should not be omitted -->` which makes it possible to leave out the braces around the continue. There's over a hundred location in Boogie where the braces are currently skipped.

I suggest we use the `StyleCop.ruleset` file as a guideline for styling. If we want to enforce curly braces everywhere, let's create a PR for that, where we update `StyleCop.ruleset` and all existing wrong cases.